fusion_bench/utils/json.py

```
@@ -2,6 +2,8 @@
 from pathlib import Path
 from typing import TYPE_CHECKING, Any, Union
 
+from fusion_bench.utils.validation import validate_file_exists, validate_path_exists
```

Flake8 (7.3.0)

[error] 'fusion_bench.utils.validation.validate_path_exists' imported but unused

(F401)